doc/system-telemetry/process-docker-stats.md

```diff
@@ -0,0 +1,84 @@
+# Process and docker stats availability thorugh telemetry agent
+
+## Revision
+
+| Rev | Date     | Author      | Change Description |
+|:---:|:--------:|:-----------:|--------------------|
+| 0.1 | 09/12/19 | Pradnya Mohite | Initial version    |
+
+## Scope
+-Enable sonic streaming telemetry agent to send Process and docker stats data
+
+### Enable sonic streaming telemetry agent to send Process and docker stats data
+
+##### Part 1 
+For 1st part, Daemon code will be added under sonic-buildimage/files/image_config.  A Daemon will start when OS starts and upload Process and docker stats data every 2 mins to state-DB. Details of CLI and state-DB given below. 
+
+##### Part 2
+From state-DB data need to be available via telemetry agent
+
+##### CLI output  and corresponding structure in state-DB for process and docket stats
+
+###### Process stats
+
+$ ps aux 
```

Comment:
in the worst case, there might be too many processes. It is better to have some limit, i am think should be limit to 1024 largest mem consumption processes and 1024 largest cpu consumption processes.

Reply:
Agree. I'll look for ways to get top 1024 processes only.